### DISCUSSION TITLE
New Page Transition feature - Disable for specific elements?

### DISCUSSION THREAD

**Discussion by noelillinger** [elementor/elementor#18134] [Developers Community]
Hey guys! 

I started using the page transition feature, really loving it. 

My problem though, it triggers the transition any time a button is clicked. On my Free Wallpaper page, where users can download images: https://noelpixels.com/free-wallpaper - anytime a user clicks download, the transition kicks in and obviously won't stop running.
The download is started using PHP - The plugin I use for it is the Simple Download Manager.

Is there a way to filter this page, or the specific button element via a code snippet?

Thank you very much for your help!

Best, Noel

**Comment by GerardPerret** [comment]
Hi,
Same issue with an image gallery litghbox used to display, would be nice to disable transition on elements in a page
ATM when you click on an image it's starting transition and lightbox does not come.
